packages/flutter/lib/src/cupertino/tab_scaffold.dart

```diff
@@ -15,6 +87,11 @@ import 'theme.dart';
 /// will automatically listen to the provided [CupertinoTabBar]'s tap callbacks
 /// to change the active tab.
 ///
+/// A [controller] can be used to provide an initialy selected tab index and manage
+/// subsequent tab changes. If a [CupertinoTabController] is provided, it will
```

Comment:
`If a [CupertinoTabController] is provided, it will always be used by this [CupertinoTabScaffold]` this sentence doesn't add much value. Just say an internal CupertinoTabController will be created if none is provided.